eventually/task/models.py

```diff
@@ -201,7 +203,7 @@ def delete_by_id(task_id):
             task.delete()
             return True
         except (Task.DoesNotExist, AttributeError):
-            pass
+            LOGGER.error('Certai task does not deleted')

```

Comment:
Certain)

Reply:
Mb u should output the reason of unseccessful deletion( like "task doesn't exist')